tests/core/data/test_generic_data_node.py

```diff
@@ -53,9 +55,11 @@ class TestGenericDataNode:
     data = list(range(10))
 
     def test_create(self):
```

Comment:
This test function is too long. Can we take the opportunity to split it into 4 test functions?